compiler/codegen/OMRCodeGenPhaseFunctionTable.hpp

```diff
@@ -26,6 +26,7 @@
    TR::CodeGenPhase::performReserveCodeCachePhase,                                           //ReserveCodeCachePhase
    TR::CodeGenPhase::performLowerTreesPhase,                                                 //LowerTreesPhase
    TR::CodeGenPhase::performSetupForInstructionSelectionPhase,                               //SetupForInstructionSelectionPhase
+   TR::CodeGenPhase::performUncommonCallConstNodesPhase,                                     //UncommonCallConstNodesPhase
```

Comment:
I just moved this phase before the `SetupForInstructonSelection` phase. 

As an aside, I do notice that there's a `RemoveUnusedLocalsPhase` stuck between the setup instruction selection and instruction selection phase ( in `trj9\codegen\CodeGenPhaseToPerform.hpp`). Is this intended?